global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-alpha.1.21613.8"
+    "version": "7.0.100-alpha.1.21617.5"
```

Comment:
changes here will conflict w/ #39127 
Suggest waiting for that to get in and handling the conflict

Reply:
Sure, thanks!